solutions/usaco-526.mdx

```diff
@@ -13,6 +13,30 @@ author: Brad Ma
 
 <LanguageSection>
 
+<PySection>
+
+```python
+import sys
+sys.stdin = open("censor.in", "r")
+sys.stdout = open("censor.out", "w")
+
+def main():
+	s = input().strip()
+	t = input().strip()
+	final = ""
+	# Add each character to the final string
+	for char in s:
+		final += char
+		# If the last len(t) characters is just t, then remove the last len(t) characters
```

Comment:
this is around 80 chars?

Reply:
Sorry, I don't quite understand what you mean. What I meant was that if the new string ends with t, we remove t from the end. I edited this comment in a new commit, so hopefully it makes more sense.

Reply:
lines should be like under 80 chars
```py
"""
to split a comment use a multiline
string like this
"""
```

Reply:
Oh, I understand what you mean now. I fixed this in commit 16ed191.